index.d.ts

```diff
@@ -2,6 +2,26 @@
 import File = require('vinyl');
 import * as globStream from 'glob-stream';
 import * as through from 'through2';
+import {EventEmitter} from 'events';
+
+export interface WatchOptions {
+    interval?: number;
+    debounceDelay?: number;
+    cwd?: string;
+    maxListeners?: Function;
+}
+
+export interface OutEvent {
+    type: any;
+    path: any;
+    old: any;
```

Comment:
Can we type this interface correctly? I managed to follow the code to https://github.com/shama/gaze. So `path` is a `string`, `old` is optional and a `string` and `type` should be `'added' | 'changed' | 'deleted'`.